src/client/common/application/types.ts

```diff
@@ -1093,7 +1093,10 @@ export interface IWebview {
 
 // Wraps the VS Code webview view
 export const IWebviewView = Symbol('IWebviewView');
-export interface IWebviewView extends IWebview {}
+export interface IWebviewView extends IWebview {
+    readonly onDidChangeVisiblity: Event<void>;
```

Comment:
vscode webviewview doesn't handle visibility the same as vscode webviewpanel, so this is handled specific to the IWebviewView interface.